src/encoded/tests/test_indexing.py

```diff
@@ -24,21 +24,23 @@
     build_index_record,
     compare_against_existing_mapping
 )
-from snovault.elasticsearch.interfaces import INDEXER_QUEUE
 from snovault.elasticsearch.indexer_utils import get_namespaced_index
```

Comment:
Most of the changes in this file are about introducing es_ apps instead of regular ones because of workbook fixtures being phased out in the way @willronchetti had done in `cgap-portal`. There are one or two other changes in this file unrelated to that which I've tried to call out.